.github/config.yml

```diff
@@ -0,0 +1,36 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats 🎉 for opening your very first issue in this project.AnitaB.org is an inclusive community, commited to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-web/blob/develop/CODE_OF_CONDUCT.md).🙌
+  You may submit a PR if you like! If you want to report a bug🐞 please follow our [Issue Template](https://github.com/anitab-org/.github/tree/main/.github/ISSUE_TEMPLATE). Also make sure you include steps to reproduce it and be patient while we get back to you.😄
+
+
+   Feel free to join us on [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/).💖 We have different streams for each active repository for discussions.✨ Hope you have a great time there!😄
+
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+
+
+ Thank you and congrats🎉 for opening your first pull request.✨ AnitaB.org is an inclusive community, commited to creating a safe and positive environment.🌸Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-web/blob/develop/CODE_OF_CONDUCT.md) and [Contribution Guidelines](https://github.com/anitab-org/open-source-programs-web/blob/develop/.github/CONTRIBUTING.md).🙌.We will get back to you as soon as we can.😄
+
+
+  Feel free to join us on [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/).💖 We have different streams for each active repository for discussions.✨ Hope you have a great time there!😄
+  
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+
+
+firstPRMergeComment: >
+  Congrats on merging your first pull request! 🎉🎉🎉
```

Comment:
Also you can add this line `We here at AnitaB.org are proud of you!` here refer [this](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/config.yml)